### PR TITLE
Extract mask variable

### PR DESCRIPTION
**Pull request by salhus**
This PR helps in extracting the mask variables. This PR introduces a new function in the following directory,
*#\source\functions\simulink\mask*

To use this function,
- Open the WEC-Sim model where the mask to be extracted resides.
- In the argument enter the address for the mask including the Simulink model name. 
- For example: mask =  ExtractMaskVariables('RM3/Float')

### CODE REVIEW

**Comment by salhus** [comment]
Hi @nathanmtom ,
I added the relevant documentation to the PR. This should be ready to merge.

File: docs/developer/advanced_features.rst [hunk 1]
```diff
@@ -18,4 +18,26 @@ Morison Element
...
+to programmatically set block parameters by being able to access the unique tags
+that Simulink uses for a particular block parameter. This is also useful for the code
+initialization of Simulink mask blocks.
+The function :code:`ExtractMaskVariables()` 
+located in :code:`source\functions\simulink\mask`, can be used to extract the block
+parameters in following steps,
+
+* Open the pertinent Simulink model,
+* Run the function with the address of the block being probed as the argument,
+* Explore the :code:`mask' data structure in the MATLAB workspace.
```
Comment (H0R5E) [inline docs/developer/advanced_features.rst:37]: @salhus, `mask` needs to be delimited by two `` ` ``s, i.e. `` :code:`mask` `` not `` :code:`mask' ``.

Comment (H0R5E) [reply docs/developer/advanced_features.rst:37]: From [this line](https://github.com/WEC-Sim/WEC-Sim/pull/958/checks#step:5:41) in the build logs.

Comment (salhus) [reply docs/developer/advanced_features.rst]: @H0R5E Thanks so much Matthew! I appreciate the help!

**Comment by nathanmtom** [comment]
The checks are now passing and will move forward with merging.

**Pull request merged**